notebooks/examples/quickstart_in_5_minutes.ipynb

```diff
@@ -0,0 +1,2112 @@
+{
```

Comment:
<p>"This notebook" -&gt; "This guide/documentation" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>